docs/extensibility/extend-rest-api-update-cart.md

```diff
@@ -0,0 +1,90 @@
+# Updating the cart with the Store API
+If you're an extension developer, and your extension does some server-side processing as a result of some client-side
+input, then you may want to use the `cart/extensions` endpoint to signal to
+the Store API that some processing needs to take place.
```

Comment:
I'm struggling to understand this part. As a plugin developer, I fail to understand where my code would fall into this, and what would be Cart/Checkout blocks responsibility.

Reply:
I think the new format may help to illustrate the problem a bit better. What do you think?